pandas/tests/scalar/timedelta/test_timedelta.py

```diff
@@ -233,6 +233,11 @@ def check(value):
         assert tup.microseconds == 999
         assert tup.nanoseconds == 0
 
+    def test_iso_convertion(self):
```

Comment:
Minor typo of the word `conversion` if you care to fix up

Reply:
Should I push a fix for it?

Reply:
Go for it